### DISCUSSION TITLE
commenting step/line in scenario in serenity 3.3.0 is not working

### DISCUSSION THREAD

**Discussion by Nagaraj-Lagashetty** [serenity-is/Serenity#7029] [Help]
commenting any perticular step in scenario in serenity 3.3.0 is making entire scenario as ignored in report. which was not the case in 2.3.0 serenity.
!-- is used to comment th step in feature file.
After updating serenitycore version from 2.3.0 to 3.3.0 behaviour mentioned in section 6.1.3 in below serenity doc is not working as expected. https://serenity-bdd.info/docs/serenity/

**Comment by VictorTomaili** [comment]
Serenity-bdd is not related our serenity platform. They are different projects with different names.
